utils/generatePlayersforTesting.js

```diff
@@ -0,0 +1,26 @@
+import {addPlayer}  from '../app/playersinscription/addPlayerDialog.js';
+import {isRankDuplicated}  from '../app/playersinscription/addPlayerBusiness.js';
```

Comment:
utils should not depend on app-specific modules. I'd expect utils to be reusable, generic logic.